src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedLanguageServerDefinition.java

```diff
@@ -69,6 +69,26 @@ public UserDefinedLanguageServerDefinition(@NotNull String id,
         this.clientConfigurationContent = clientConfigurationContent;
     }
 
+    // Backward-compatible signature for clients calling without client configuration content
+    public UserDefinedLanguageServerDefinition(@NotNull String id,

```

Comment:
The comment says it all, but because this is part of the public API -- and is in fact used by my own plugin to assist with language server config -- the original constructor signature must be restored/retained, passing a null value for `clientConfiguraitonContent`.